src/protos/io/defang/v1/fabric.proto

```diff
@@ -197,7 +195,7 @@ message LogEntry {
 message TailResponse {
   reserved 1; // was "string raw"
   repeated LogEntry entries = 2;
-  string service = 3;
+  repeated string services = 3;
```

Comment:
I don't think this is correct. See my comment on the other PR. Talk to Edward: I think he wants to deprecate this field.

Reply:
I am adding service, etag and host to every LogEntry, as an override to the default values from the TailResponse, see; [#457](https://github.com/DefangLabs/defang/pull/457/files#diff-3f2d6a56e3de05f109a5cb18dc97d04706e787d01fe2a799d97b71e7b1861422R195-R197)